Gems/EMotionFX/Code/EMotionFX/Source/Importer/ActorFileFormat.h

```diff
@@ -99,6 +99,24 @@ namespace EMotionFX
             // string : the name of the actor
         };
 
+        // (aligned)
+        struct Actor_Info4
+        {
+            uint64  mNumLODs;               // the number of level of details
+            uint64  mMotionExtractionNodeIndex;// the node number of the trajectory node used for motion extraction
+            uint64  mRetargetRootNodeIndex; // the retargeting root node index, most likely pointing to the hip or pelvis or MCORE_INVALIDINDEX32 when not set
+            uint8   mUnitType;              // maps to EMotionFX::EUnitType
+            uint8   mExporterHighVersion;
+            uint8   mExporterLowVersion;
+            uint8   mOptimizeSkeleton;
+
+            // followed by:
+            // string : source application (e.g. "3ds Max 2011", "Maya 2011")
+            // string : original filename of the 3dsMax/Maya file
+            // string : compilation date of the exporter
+            // string : the name of the actor
+        };
```

Comment:
Do we need this really?? Can't we just use the already existing version and convert the values to 64-bit?

Reply:
I can try to remove this. I think I was probably being pedantic, that the `Actor` type now stores up to 64bit LODs, and the `Skeleton` has up to 2^64-1 nodes, so the motion extraction index and retarget root node index also have to support those index values. But if someone is making an actor with more than 4294967295 joints, we probably have other problems.

Reply:
Haha yeah ;) Ok, let's remove it and keep the old version.

Reply:
I've removed the new Actor version from this PR.